dockertest/config.py

```diff
@@ -247,6 +244,7 @@ class ConfigDict(MutableMapping):
     def __init__(self, section, defaults=None, *args, **dargs):
         self._config_section = ConfigSection(defaults=defaults,
                                              section=section)
+        # pylint: disable=E1101
         super(ConfigDict, self).__init__(*args, **dargs)
```

Comment:
E1101:250,8: ConfigDict.\_\_init\_\_: Super of 'ConfigDict' has no '\_\_init\_\_' member